src/main/java/ChatReques11.java

```diff
@@ -0,0 +1,27 @@
+import lombok.*;
+
+import java.util.List;
+
+/**
+ * @author madhankumar
+ */
+@Setter
+@Getter
+@ToString
+@Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+public class ChatReques11 {
```

Comment:
:information_source: Class name should follow Java naming conventions (ChatReques11 should be ChatRequest) (Severity: Minor)